spec/beaker_test_helpers.rb

```diff
@@ -0,0 +1,17 @@
+# These are specifically to mock Beaker methods necessary for testing
+# that will be available during runtime because this is never run separate
+# from Beaker itself.
+#
+# Including Beaker as a dependency would not work as a solution to this issue,
+# since that would make a cycle in the dependency graph, at least until
+# Beaker 3.0 happens and this is no longer a dependency of Beaker's.
+module BeakerTestHelpers
```

Comment:
doh, I didn't realize it was in the context of the spec folder. That's totally sufficient.